app/permission/rest.py

```diff
@@ -10,17 +14,12 @@
 
 @permission.route('', methods=['GET'])
 def get_permissions():
-    data, errors = permission_schema.dump(
-        permission_dao.get_query(filter_by_dict=request.args), many=True)
-    if errors:
-        abort(500, errors)
+    data = permission_schema.dump(permission_dao.get_query(filter_by_dict=request.args), many=True).data
```

Comment:
this'll now 400 instead of 500 - is that ok? i assume we 500'd for a reason, but not sure what that is/was